experimental/packages/api-logs/src/ProxyLoggerProvider.ts

```diff
@@ -29,23 +28,19 @@ export class ProxyLoggerProvider implements LoggerProvider {
     options?: LoggerOptions | undefined
   ): Logger {
     return (
-      this.getDelegateLogger(name, version, options) ??
+      this._getDelegateLogger(name, version, options) ??
       new ProxyLogger(this, name, version, options)
     );
   }
 
-  getDelegate(): LoggerProvider {
```

Comment:
Looks like this one was added for testing purposes only, why removing the tests?

Reply:
Oh I didn't know that we add methods for testing purposes only. I can make it private then and add it back to the tests.

Reply:
I'm not sure about the original purpose, it may be that this logic was duplicated from ProxyTracerProvider, but I can see is only used in the project for testing :)